### PR TITLE
CI：Speed up CI jobs with parallel 

### PR DESCRIPTION
**Pull request by alcorj-mizar**

### CODE REVIEW

**Comment by k82cn** [comment]
LGTM to me overall :)

File: .travis.yml [hunk 1]
```diff
@@ -39,8 +45,8 @@ jobs:
       script:
         - make vcctl
         - make images
-        - make e2e-test-job
-    - stage: E2E Scheduling
+        - travis_retry make e2e-test-jobp
```
Comment (hzxuzhonghu) [inline .travis.yml:48]: `travis_retry` what does this mean

File: Makefile [hunk 2]
```diff
@@ -75,16 +75,22 @@ manifests: controller-gen
...
 	./hack/run-e2e-kind.sh
 
-e2e-test-scheduling:
-	E2E_TYPE=SCHEDULING ./hack/run-e2e-kind.sh
+e2e-test-schedulingbase:
+	E2E_TYPE=SCHEDULINGBASE ./hack/run-e2e-kind.sh
 
-e2e-test-job:
-	E2E_TYPE=JOB ./hack/run-e2e-kind.sh
+e2e-test-schedulingaction:
```
Comment (hzxuzhonghu) [inline Makefile:86]: What's the diff between schedulingbase and schedulingaction

File: .travis.yml (hunk 1 above)
Comment (alcorj-mizar) [reply .travis.yml:48]: If make e2e-test-jobp failed, it will retry for 3 times.

File: Makefile (hunk 2 above)
Comment (alcorj-mizar) [reply Makefile:86]: schedulingbase is the basic scheduling test, schedulingaction is just include action plugin.

File: .travis.yml (hunk 1 above)
Comment (alcorj-mizar) [reply .travis.yml:48]: It is a small probability if kind create cluster will failed of write configuration, and this command will retry.

**Pull request merged**